src/components/ProductDetail/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useParamsTypes } from "./types";
+import useFetch from "../../hooks/useFetch/useFetch";
+import styles from "./styles.module.scss";
+import useCart from "../../hooks/useCart";
+import ShoppingCart from "../ShoppingCart";
+import Header from "../Header";
+import Cart from "../ShoppingCart/Cart";
+import DefaultButton from "../button/DefaultButton";
+import Footer from "../Footer";
+
+const url = "https://fakestoreapi.com/products?limit=25";
```

Comment:
boleh pakai api product detail aja,
![image](https://github.com/elbertwidjaja/Dummy_Project_1_KF/assets/75367447/ee4cd6cd-b391-46ce-978a-bce8b54b38a0)

umumnya sudah di siapkan juga dari tim be untuk data detail terutama dari product jadi kita ga perlu filter lagi